esileapclient/tests/unit/common/test_utils.py

```diff
@@ -0,0 +1,73 @@
+import unittest
+from esileapclient.common import utils
+
+
+class TestUtils(unittest.TestCase):
+
+    def setUp(self):
+        self.nodes = [
+            {'properties': {'cpus': '40', 'memory_mb': '131072'}},
+            {'properties': {'cpus': '80', 'memory_mb': '262144'}},
+            {'properties': {'cpus': '20', 'memory_mb': '65536'}},
+        ]
+        self.parser = utils.PropertyFilterParser()
+
+    def test_convert_value(self):
+        self.assertEqual(self.parser.convert_value('10'), 10)
+        self.assertEqual(self.parser.convert_value('10.5'), 10.5)
+
+    def test_parse_property_filter(self):
+        key, op, value = self.parser.parse_property_filter('cpus>=40')
+        self.assertEqual(key, 'cpus')
+        self.assertEqual(op, utils.PropertyFilterParser.OPS['>='])
+        self.assertEqual(value, 40)
+
+        key, op, value = self.parser.parse_property_filter('memory_mb<=131072')
+        self.assertEqual(key, 'memory_mb')
+        self.assertEqual(op, utils.PropertyFilterParser.OPS['<='])
+        self.assertEqual(value, 131072)
+
+        with self.assertRaises(ValueError):
+            self.parser.parse_property_filter('invalid_filter')
```

Comment:
Is it possible to also test for an invalid operator?

Reply:
`>' not supported between instances of 'int' and 'str'` or 

```
Error parsing property filter 'cpus!40': Invalid property filter format: cpus!40
Invalid property filter format: 

> cpus!40
```

 Without filters it prints this kind of message but definitely. I will include the test case for that.